tools/Generate-CodepointWidthsFromUCD.ps1

```diff
@@ -0,0 +1,252 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+#Requires -Version 7
+# (we use the null coalescing operator)
+
+[Diagnostics.CodeAnalysis.SuppressMessageAttribute('PSAvoidUsingPositionalParameters', '')]
+[Diagnostics.CodeAnalysis.SuppressMessageAttribute('PSUseProcessBlockForPipelineCommand', '')]
+[CmdletBinding()]

```

Comment:
Please leave help option and/or examples of how to run this.

Reply:
I added a giant block comment instead. Will that be sufficient?

Reply:
I could write a whole Help, but... i'd prefer to make this part of the openconsole module before I do that, and I don't want to do that right now.

Reply:
good enough for now. this helps